components/organisms/Repositories/repositories.tsx

```diff
@@ -55,7 +55,13 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
         "/hub/insights/new"
       );
     } else {
-      signIn({ provider: "github" });
+      const reposIds = selectedRepos.map((repo) => repo.id);
```

Comment:
nit: pass into the url a comma separated string 
```suggestion
      const reposIds = selectedRepos.map((repo) => repo.id).join(",")
```